lib/src/core.rs

```diff
@@ -1082,6 +1082,7 @@ pub struct Erc721Properties {
     pub license: Option<String>,
     pub screenshots: Option<Vec<String>>,
     pub wit_version: Option<(u32, u32, u32)>,
+    pub dependencies: Vec<String>,
```

Comment:
Let's make this an `Option<Vec<String>>` and please add the description of this field to the docstring for this type

Reply:
What is the purpose of having it be an `Option` when an empty vector accomplishes the same thing? JSON parsing optional field?

Reply:
Changed in 3194b03